src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -215,6 +228,22 @@ public void commit() {
     successfulCommitTime = time.milliseconds();
   }
 
+  /**
+   * Callback that is called when a record is committed.
+   */
+  @Override
+  public void commitRecord(SourceRecord record, RecordMetadata metadata) {
+      //
+      // record.kafkaPartition() is null when 1:1 partition mapping is not enabled.
+      //
+      try {
+        long latency = System.currentTimeMillis() - metadata.timestamp();
+        mirrorJmxReporter.recordMirrorLatency(record.topic(), latency);
+      } catch (Exception exception){
+        logger.error(exception.getMessage(), exception);
```

Comment:
You're effectively swallowing the exception here, which is rarely a good idea. Is this needed at all given the standard handler will log the exception? Can you re-throw the exception?

Reply:
Commit is called [here](https://github.com/apache/kafka/blob/2.6.1/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java#L456-L462) so I guess there isn't a need to log the exception in this code. I will remove.